app/models/academies_api/base_api_model.rb

```diff
@@ -0,0 +1,22 @@
+class AcademiesApi::BaseApiModel
+  include ActiveModel::Serializers::JSON
+
+  class AttributeMapMissingError < RuntimeError; end
```

Comment:
I've not seen an error like this inherit from RunTimeError, was there a reson you chose to?

Reply:
I think this is standard in the other app I have worked on, and it seems more semantically correct? 
70 examples in the DfE Github 👇 
https://github.com/search?q=org%3ADFE-Digital+%3C+runtimeerror&type=code